prisma/schema.prisma

```diff
@@ -0,0 +1,31 @@
+datasource db {
```

Comment:
the following lines need to be generated at the top of the file as well

```
generator client {
  provider = "prisma-client-js"
  previewFeatures = ["postgresqlExtensions", "views"]
}
```

Reply:
generator client takes value in this format: `createGenerator(name: string, provider: string, output?: string | null | undefined, binaryTargets?: string[] | undefined): Generator`.

Reply:
this is the type of output that can be generated: generator client {
  provider      = "prisma-client-js"
  output        = ""
  binaryTargets = []
}

Reply:
can you check and find the equivalents of these in the prisma docs? in case these in backward incompatible breaking change in prisma dsl layer, let's fork and update the dsl package itself.